app/util.js

```diff
@@ -10,6 +11,16 @@ exports.passwordEncryption = user =>
       ...user,
       password
     }))
-    .catch(err => {
-      throw errors.encryptionError(`${err.message}`);
+    .catch(() => {
```

Comment:
You can do `.catch(errors.encryptionError)`. You pass implicitly the parameters received by catch and return the function. Its similar to do `.catch(err => errors.encryptionError(err))`.